apps/tup-ui/src/components/auth/LoginComponent/LoginComponent.module.css

```diff
@@ -2,37 +2,49 @@
 @import url('@tacc/core-styles/dist/settings/space.css');
 @import url('@tacc/core-styles/dist/settings/font.css');
 
+@import url('@tacc/core-styles/dist/elements/form.cms.css');
```

Comment:
ℹ️ I base the login component off [styles for CMS form](https://github.com/TACC/tup-ui/pull/47/files#diff-ec93cc1b7662a03a710ddc0a4e0730ad06c9ba1f94d13b9d2711f8fc99b99b24) (which I based off the design for the login form).